### PR TITLE
[Emote Utility] Fixed to work with the API-refactor

### PR DESCRIPTION
**Pull request by 21Joakim**

### CODE REVIEW

**Comment by davidcralph** [comment]
shouldn't this plugin be moved into its own repo now?

File: src/Powercord/plugins/pc-emojiUtility/components/Settings.jsx [hunk 1]
```diff
@@ -2,26 +2,61 @@ const { existsSync, lstatSync } = require('fs');
...
 
-const { getGuild } = getModule([ 'getGuild' ]);
-const { getSortedGuilds } = getModule([ 'getSortedGuilds' ]);
+let getGuild, getSortedGuilds;
 
 module.exports = class EmojiUtilitySettings extends React.Component {
   constructor (props) {
     super();
 
     this.settings = props.settings;
```
Comment (cyyynthia) [inline src/Powercord/plugins/pc-emojiUtility/components/Settings.jsx]: Using the plugins's settings object in the settings component is deprecated

File: src/Powercord/plugins/pc-emojiUtility/components/Settings.jsx [hunk 2]
```diff
@@ -2,26 +2,61 @@ const { existsSync, lstatSync } = require('fs');
...
-      isCloneIdValid: props.settings.get('defaultCloneId') ? getGuild(props.settings.get('defaultCloneId')) : true,
-      initialCloneIdValue: props.settings.get('defaultCloneId') || null,
+    if (getGuild && getSortedGuilds) {
+      this._setState(false);
+    }
+  }
+
+  _setState (update = true) {
+    const state = {
+      useEmbeds: this.settings.get('useEmbeds', false),
```
Comment (cyyynthia) [inline src/Powercord/plugins/pc-emojiUtility/components/Settings.jsx]: use this.props.getSetting instead

**Review by cyyynthia — CHANGES_REQUESTED** [review]
You also used `this.set` in the settings component, and this is deprecated as well. use `this.props.updateSetting(setting, newValue)` and `this.props.toggleSetting(setting)` instead

**Comment by cyyynthia** [comment]
> 
> 
> shouldn't this plugin be moved into its own repo now?

It'll get moved with most of other internal plugins once the Plugins tab is finished

**Review by cyyynthia — APPROVED** [review]

**Pull request merged**